src/main/java/com/google/api/codegen/transformer/SampleTransformer.java

```diff
@@ -251,6 +255,18 @@ private MethodSampleView generateSample(
                     initCodeContext.cloneWithEmptySymbolTable())); // to avoid symbol collision
     ImportSectionView sampleImportSectionView =
         sampleImportTransformer().generateImportSection(methodContext);
+    SampleFunctionDocView sampleFunctionDocView =
+        SampleFunctionDocView.newBuilder()
+            .paramDocLines(paramDocLines(methodContext, initCodeView))
```

Comment:
nit: I suggest passing `DOC_LINE_MAX_WIDTH` to `paramDocLines()` so both uses of the constant are within the same function. I find the asymmetry troubling :man_shrugging: